Solutions/NXLogAixAudit/Data Connectors/NXLogAixAudit.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "NXLogAixAudit",
+    "title": "NXLog AIX Audit",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog [AIX Audit](https://nxlog.co/documentation/nxlog-user-guide/im_aixaudit.html) data connector uses the AIX Audit subsystem to read events directly from the kernel for capturing audit events on the AIX platform. This REST API connector can efficiently export AIX Audit events to Azure Sentinel in real time.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**NXLog_parsed_AIX_Audit_view**](https://aka.ms/sentinel-nxlogaixaudit-parser) which is deployed with the Azure Sentinel Solution.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "NXLog_parsed_AIX_Audit_view",
+            "baseQuery": "NXLog_parsed_AIX_Audit_view"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "AIX Audit event type distribution",
+            "query": "NXLog_parsed_AIX_Audit_view\n| summarize count() by EventType\n| render piechart title=\"AIX Audit event type distributon\""
+        },
+        {
+            "description" : "Highest event per second (EPS) AIX Audit event types",
+            "query" : "NXLog_parsed_AIX_Audit_view\n where EventEndTime >  todatetime('2021-09-09')\n| summarize EPS=count() by bin(EventEndTime, 1s), EventType\n| sort by EPS, EventType, EventEndTime\n| take 5\n| render columnchart title=\"Highest event per second (EPS) event types\""

```

Comment:
This query is failing because it is missing **|** before where clause.

Reply:
This should be resolved with [d838206](https://github.com/Azure/Azure-Sentinel/pull/2968/commits/d8382068862ec8b002dc5c87010ba57b4dd1364e).

Thanks!